### ISSUE TITLE
HtmlTestRunner cannot be directly used with Django DiscoverRunner

### ISSUE THREAD

**Issue by marmistrz** [oldani/HtmlTestRunner#12]
* HtmlTestRunner version: 1.1.0
* Python version: 3.6.1
* Operating System: Arch Linux

### Description

Because `output` is a required parameter, `HtmlTestRunner` cannot be used with Django.
### What I Did

The way I managed to make Django use `HtmlTestRunner` was to make a wrapper class.
```
from django.test.runner import DiscoverRunner
from HtmlTestRunner import HTMLTestRunner


class MyRunner(HTMLTestRunner):
    def __init__(self, **kwargs):
        super().__init__(output="", **kwargs)


DiscoverRunner.test_runner = MyRunner
```

I can contribute the fix if we get down to what it should be. One solution would be to make the `output` parameter optional. Another: make a class factory function, something like

```
def makeclass(output: str) -> Type[HTMLTestRunner]:
    class MyRunner(HTMLTestRunner):
        def __init__(self, **kwargs):
            super().__init__(output=output, **kwargs)

    return MyRunner
```

What do you think?

**Comment by oldani** [comment]
We could make this optional, so if output  not specified results will be just place in the reports dir

**Comment by marmistrz** [comment]
For me, for example, it would be useful to keep it always in test_report.html. After all, I always need only the latest test report.

**Comment by oldani** [comment]
This has been fixed and released on version `1.2`. Thanks @JamesMTSloan
